src/Microsoft.Identity.Web/WebApiExtensions/MicrosoftIdentityWebApiAuthenticationBuilderExtensions.cs

```diff
@@ -79,7 +79,7 @@ public static class MicrosoftIdentityWebApiAuthenticationBuilderExtensions
         {
             _ = Throws.IfNull(configurationSection);
             _ = Throws.IfNull(builder);
-
+            
             AddMicrosoftIdentityWebApiImplementation(
                 builder,
                 options => configurationSection.Bind(options),
```

Comment:
here and 86, 83, 94, we can remove the lambda:
`options => configurationSection.Bind(options),` -> `configurationSection.Bind(options),`